reporter/src/main/kotlin/reporters/NoticeTemplateReporter.kt

```diff
@@ -225,15 +225,23 @@ class NoticeTemplateReporter : Reporter {
             licenseView: LicenseView = LicenseView.ALL,
             omitExcluded: Boolean = true
         ): List<ResolvedLicense> =
-            models
-                .filter { !omitExcluded || !it.excluded }
-                .flatMap {
+            mergeLicenses(
+                models.filter { !omitExcluded || !it.excluded }.flatMap {
                     val licenses = it.license.filter(licenseView).licenses
                     if (omitExcluded) licenses.filterExcluded() else licenses
                 }
-                .groupBy { it.license }
+            )
+
+        /**
+         * Return a list of [ResolvedLicense]s where all duplicate entries for a single license in [licenses] are
+         * merged. The returned list is sorted by license identifier.
+         */
+        @Suppress("UNUSED") // This function is used in the templates.
+        fun mergeLicenses(licenses: List<ResolvedLicense>): List<ResolvedLicense> {
```

Comment:
How about ditching the button by using `=`. Cannot remember the term for it again ^